files/es/learn/accessibility/mobile/index.html

```diff
@@ -296,7 +296,7 @@ <h4 id="Mantener_los_menús_accesibles.">Mantener los menús accesibles.</h4>
 
 <p>Debido a que la pantalla es mucho más estrecha en los dispositivos móviles, es muy común utilizar consultas de medios y otras tecnologías para hacer que el menú de navegación se reduzca a un pequeño icono en la parte superior de la pantalla, que se puede presionar para mostrar el menú solo si es necesario - cuando el sitio se ve en el móvil. Esto suele representarse mediante un icono de "tres líneas horizontales" y, por lo tanto, el patrón de diseño se conoce como "menú de hamburguesas".</p>
 
-<p>Al implementar un menú de este tipo, debe asegurarse de que el control para revelarlo sea accesible mediante los mecanismos de control apropiados (normalmente, toque para dispositivos móviles), como se explica en {{ anch("Control mechanisms" )}}, y que el resto La página se aleja o se oculta de alguna manera mientras se accede al menú, para evitar confusiones al navegar.</p>
+<p>Al implementar un menú de este tipo, debe asegurarse de que el control para revelarlo sea accesible mediante los mecanismos de control apropiados (normalmente, toque para dispositivos móviles), como se explica en <a href="#Control mechanisms">Control mechanisms</a>, y que el resto La página se aleja o se oculta de alguna manera mientras se accede al menú, para evitar confusiones al navegar.</p>
```

Comment:
Is it right for this anchor?

```suggestion
<p>Al implementar un menú de este tipo, debe asegurarse de que el control para revelarlo sea accesible mediante los mecanismos de control apropiados (normalmente, toque para dispositivos móviles), como se explica en <a href="#mecanismos_de_control">Control mechanisms</a>, y que el resto La página se aleja o se oculta de alguna manera mientras se accede al menú, para evitar confusiones al navegar.</p>
```

![image](https://user-images.githubusercontent.com/15844309/169923239-8f485b3c-fae6-4988-b2bf-150fdf88fc91.png)

Reply:
Thank you for pointing this out, could you please open another PR changing this? thanks!

Reply:
I've fixed this in #5792.

Reply:
Thank you!